bank2ynab.conf

```diff
@@ -13,20 +13,21 @@ Source CSV Delimiter = ,
 Header Rows = 1
 Footer Rows = 0
 Input Columns = Date,Payee,Outflow,Inflow,Running Balance
-# (see https://docs.python.org/2/library/datetime.html#id1 for date format strings)
+# (see https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes for date format strings)
```

Comment:
This is weird - `develop` already uses this Python 3 documentation link, why is it showing as a change?